test/unit/uk/gov/hmrc/thirdpartyapplication/controllers/GatekeeperControllerSpec.scala

```diff
@@ -193,29 +198,27 @@ class GatekeeperControllerSpec extends UnitSpec with ScalaFutures with MockitoSu
     val gatekeeperUserId = "big.boss.gatekeeper"
     val rejectUpliftRequest = RejectUpliftRequest(gatekeeperUserId, "Test error")
     val testReq = request.withBody(Json.toJson(rejectUpliftRequest)).withHeaders(authTokenHeader)
-    "return unauthorised when the user is not authorised" in new Setup {
-      when(mockAuthConnector.authorized(any[AuthRole])(any[HeaderCarrier])).thenReturn(successful(false))
+    "return forbidden when the user is not authorised" in new Setup {
+      givenUserIsNotAuthenticated(underTest)
 
       val result = await(underTest.rejectUplift(applicationId)(testReq))
 
       verifyZeroInteractions(mockGatekeeperService)
-      verifyUnauthorized(result)
+      verifyForbidden(result)
     }
 
     "successfully reject uplift when user is authorised" in new Setup {
-      val hcArgCaptor = ArgumentCaptor.forClass(classOf[HeaderCarrier])
-      when(mockAuthConnector.authorized(any[AuthRole])(hcArgCaptor.capture())).thenReturn(successful(true))
+      givenUserIsAuthenticated(underTest)
 
       when(mockGatekeeperService.rejectUplift(applicationId, rejectUpliftRequest)).thenReturn(UpliftRejected)
 
       val result = await(underTest.rejectUplift(applicationId)(testReq))
 
-      hcArgCaptor.getValue.authorization.get.value shouldBe "authorizationToken"
       status(result) shouldBe 204
```

Comment:
Can we change this 204 to NO_CONTENT? And any other similar "magic number" occurences.

Reply:
There are loads of magic numbers in this file. Wary of going down the rabbit hole of cleaning up untouched files.